packages/cli/src/Interfaces.ts

```diff
@@ -337,6 +337,7 @@ export interface IN8nUISettings {
 		[key: string]: string | number | undefined;
 	};
 	versionNotifications: IVersionNotificationSettings;
+	instanceId?: string;
```

Comment:
since we are always setting a value, this does need to be optional
```suggestion
	instanceId: string;
```